src/Type/InterfaceType/EnqueuedAsset.php

```diff
@@ -94,8 +94,8 @@ static function ( $before ) {
 						},
 					],
 					'dependencies' => [
-						'type'        => [ 'list_of' => 'EnqueuedAsset' ],
-						'description' => __( 'Dependencies needed to use this asset', 'wp-graphql' ),
+						'type'        => [ 'list_of' => 'String' ],
```

Comment:
Why drop the entire object? The handles are available via `dependencies.nodes.handle`, and since there's a dataloader + we parse from the global and not a db query there's no performance loss for the extra context.

Reply:
True, and it's breaking CI too. I honestly don't remember if there was major reason for remember, but I'll restore and see if it breaks my solutions. Hopefully, I can just put it back.